Mathlib/Combinatorics/Additive/Energy.lean

```diff
@@ -84,10 +84,7 @@ lemma mulEnergy_mono (hs : s₁ ⊆ s₂) (ht : t₁ ⊆ t₂) : Eₘ[s₁, t₁
 @[to_additive] lemma le_mulEnergy : s.card * t.card ≤ Eₘ[s, t] := by
   rw [← card_product]
   refine
-    card_le_card_of_injOn (@fun x => ((x.1, x.1), x.2, x.2)) (by
```

Comment:
Can you benchmarking this file/check locally if this is an issue?

Reply:
It was 688 heartbeats, not 882, so completely irrelevant.